d2/src/main/java/com/linkedin/d2/balancer/clients/RetryClient.java

```diff
@@ -78,13 +78,17 @@
 {
   public static final long DEFAULT_UPDATE_INTERVAL_MS = TimeUnit.SECONDS.toMillis(1);
   public static final int DEFAULT_AGGREGATED_INTERVAL_NUM = 5;
+  public static final boolean DEFAULT_REST_RETRY_ENABLED = true;
```

Comment:
Maybe `false` is a safer default in code. Factory config can default to `true`.

Reply:
Done